integration-tests/wallet-edo-deploy-having-ticket.spec.ts

```diff
@@ -11,7 +15,52 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
       done()
     })
 
-    it('Verify wallet.originate for a contract having ticket with init', async (done) => {
+    kathmandunet('Verify wallet.originate for a contract having ticket with init', async (done) => {
+      const op = await Tezos.wallet.originate({
+        code: ticketCodeProto14,
+        init: ticketStorageProto14
+      }).send();
+
+      await op.confirmation();
+      expect(op.opHash).toBeDefined();
+
+      done();
+    });
+
+    kathmandunet('Verify wallet.originate having ticket with init in JSON', async (done) => {
+      const op = await Tezos.wallet.originate({
+        code: ticketCodeProto14,
+        init: { prim: 'Pair', args: [ { prim: 'None' }, { prim: 'None' } ] }
+      }).send();
+
+      await op.confirmation();
+      expect(op.opHash).toBeDefined();
+      done();
+    });
+
+    kathmandunet('Verify wallet.originate for a contract having ticket with storage', async (done) => {
+      const op = await Tezos.wallet.originate({
+        code: ticketCodeProto14,
+        storage: {
+          '%x': null,
+          '%y': null
+        }
+      }).send();
+
+      await op.confirmation();
+      expect(op.opHash).toBeDefined();
+
+      done();
+    });
+  });
+  describe(`Test contract origination having ticket with init through wallet api using: ${rpc}`, () => {
+
+    beforeEach(async (done) => {
+      await setup();
+      done()
+    })
```

Comment:
nitpick, but missing a few semicolons here and the line above

Reply:
pushed a commit to add semicolons